GotExplorer.BLL/Services/EmailService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using GotExplorer.BLL.Options;
+using GotExplorer.BLL.Services.Interfaces;
+using Microsoft.Extensions.Options;
+using System.Net;
+using System.Net.Mail;
+
+namespace GotExplorer.BLL.Services
+{
+    public class EmailService : IEmailService
+    {
+        private readonly SmtpOptions _smtpOptions;
+
+        public EmailService(IOptions<SmtpOptions> smtpOptions)
+        {
+            _smtpOptions = smtpOptions.Value;
+        }
+
+        public void SendEmail(string toEmail, string subject, string body)
+        {
+            using (var smtpClient = new SmtpClient(_smtpOptions.Host,_smtpOptions.Port))
```

Comment:
Here are a lot of limits based on smtp sender by using gmail or other free tool 
My suggestion is to take a look on ready solutions like SendGrid